src/main/scala/tilelink/Monitor.scala

```diff
@@ -484,34 +574,74 @@ class TLMonitor(args: TLMonitorArgs, monitorDir: MonitorDirection = MonitorDirec
   }
 
   def legalizeADSource(bundle: TLBundle, edge: TLEdge) {
+    val a_size_bus_size = edge.bundle.sizeBits
```

Comment:
Theres no additional left shifting. Unless theres an additional flag that I'm missing?

Reply:
Compare with: https://github.com/chipsalliance/rocket-chip/pull/2138/files

Reply:
I fixed this